extensions/frontend/pkg/revaconfig/config.go

```diff
@@ -217,7 +218,7 @@ func FrontendConfigFromStruct(cfg *config.Config) map[string]interface{} {
 							"major":   version.Parsed().Major(),
 							"minor":   version.Parsed().Minor(),
 							"micro":   version.Parsed().Patch(),
```

Comment:
These need to have "legacy" values that match the parts of `LegacyString`, to make the response consistent.